dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowTask.java

```diff
@@ -197,6 +206,33 @@ private Map<String, Property> getParamsMap() {
 
     }
 
+    public int checkDockerHealth() throws Exception {
```

Comment:
I think it's a general method and can move into `ProcessUtils`, WDYT?

Reply:
I can't make a sound recommendation because I'm not sure if there are other components that use this function. WDYT @zhongjiajie